corehq/toggles/__init__.py

```diff
@@ -1030,14 +1030,23 @@ def _ensure_valid_randomness(randomness):
     namespaces=[NAMESPACE_DOMAIN]
 )
 
+DYNAMICALLY_UPDATE_SEARCH_RESULTS = StaticToggle(
+    'dynamically_update_search_results',
+    "In case search with split screen case search enabled, search results update when a search field is updated"
+    " without requiring the user to manually press a button to search.",
+    TAG_CUSTOM,
+    help_link='https://confluence.dimagi.com/display/USH/Split+Screen+Case+Search',
+    namespaces=[NAMESPACE_DOMAIN],
+)
+
 SPLIT_SCREEN_CASE_SEARCH = StaticToggle(
     'split_screen_case_search',
     "Split screen case search: In case search, show the search filters in a sidebar on the left and the results"
     " on the right.",
     TAG_CUSTOM,
     help_link='https://confluence.dimagi.com/display/USH/Split+Screen+Case+Search',
     namespaces=[NAMESPACE_DOMAIN],
-    parent_toggles=[SYNC_SEARCH_CASE_CLAIM]
+    parent_toggles=[SYNC_SEARCH_CASE_CLAIM, DYNAMICALLY_UPDATE_SEARCH_RESULTS]
```

Comment:
Believe Simon commented on the other PR, but I think this is backwards, right? `DYNAMICALLY_UPDATE_SEARCH_RESULTS` would probably require `SPLIT_SCREEN_CASE_SEARCH` to be enabled, not the other way around.

Reply:
resolved, and also [see comment here](https://github.com/dimagi/commcare-hq/pull/33429#issuecomment-1711711066)